school_library_decorator/student.rb

```diff
@@ -1,12 +1,15 @@
-require './person.rb'
-
+require './person'
+# rubocop:disable Style/OptionalBooleanParameter
 class Student < Person
-  def initialize(age, classroom, name = "Unknown", parent_permission = true)
+  attr_accessor :classroom
+
+  def initialize(age, classroom, name = 'Unknown', parent_permission = true)
     super(age, name, parent_permission)
     @classroom = classroom
   end
 
   def play_hooky
     "¯\(ツ)/¯"
   end
-end
\ No newline at end of file
+end
+# rubocop:enable Style/OptionalBooleanParameter
```

Comment:
- [ ] Please don't disable linter rules. Disabling rubocop might help with linter errors but these are here to help us write clean code.